apps/vaporgui/VDoubleLineEdit.cpp

```diff
@@ -0,0 +1,79 @@
+#include <string>
+#include <sstream>
+#include <iomanip>
+
+#include <QString>
+
+#include "VDoubleLineEdit.h"
+
+VDoubleLineEdit::VDoubleLineEdit( double value, bool useMenu ) : 
```

Comment:
`bool useMenu` is not used. Same for VIntLineEdit

Reply:
Thanks.  Fixed.